docs/tutorial/dbinit.rst

```diff
@@ -44,6 +44,9 @@ Within the function, you can then access :attr:`flask.g` and other things as
 you might expect.  When the script ends, the application context tears down
 and the database connection is released.
 
+.. note:: In order to use the newly created ``flask initdb`` command, the 
+    application must be re-installed using ``pip install -e .``
```

Comment:
Well, technically at its current state the commands are listed in the order they have to be typed and it is functional, when followed through linearly. But it would be nice to avoid the repetition. Which I guess is what you are getting at. But correct me if I am wrong.